models/lorenz_96/work/input.nml

```diff
@@ -92,9 +92,12 @@
 &ensemble_manager_nml
    /
 
+# the localization cutoff in this file is distributed is too
+# large for the model to converge.  to test that the model is 
+# doing a successful assimilation, change cutoff to 0.02 and rerun.
 &assim_tools_nml
    filter_kind                     = 1,
-   cutoff                          = 0.02,
+   cutoff                          = 1000000.0,
```

Comment:
```suggestion
   cutoff                          = 0.2,
```

Is this a relic from localization testing?

Reply:
all changes except the preprocess changes should be reverted.  i think it's a relic of running the workshop setup which uses different values for these things.